src/components/application_manager/rpc_plugins/sdl_rpc_plugin/test/commands/hmi/navi_audio_start_stream_request_test.cc

```diff
@@ -74,11 +75,24 @@ class AudioStartStreamRequestTest
     command_ = CreateCommand<AudioStartStreamRequest>(msg_);
   }
 
+  void UpdateMsgParams(MessageSharedPtr& message) {
+    (*message)[am::strings::params][am::strings::correlation_id] =
+        kCorrelationId;
+    (*message)[am::strings::params][am::strings::message_type] =
+        am::MessageType::kRequest;
+    (*message)[am::strings::params][am::strings::function_id] =
+        static_cast<int32_t>(hmi_apis::FunctionID::Navigation_StartAudioStream);
+    (*message)[am::strings::msg_params][am::strings::app_id] = kHmiAppId;
+  }
+
   std::pair<uint32_t, int32_t> start_stream_retry_amount_;
   MessageSharedPtr msg_;
   std::shared_ptr<AudioStartStreamRequest> command_;
+  static const std::string big_url_;
```

Comment:
@SNiukalov Why did you make it static?

Reply:
@AByzhynar 
Made it static for performance optimization.

Reply:
@SNiukalov If I understand optimization purpose in this case correctly - better to do it in C++ way:
```cpp
namespace {
const std::string kBigURL(20000u, 'a');
}
```

Reply:
@AByzhynar 
This is just alternative option